src/Symfony/Bundle/WebServerBundle/WebServer.php

```diff
@@ -167,7 +167,6 @@ private function createServerProcess(WebServerConfig $config)
 
         if (\in_array('APP_ENV', explode(',', getenv('SYMFONY_DOTENV_VARS')))) {
             $process->setEnv(['APP_ENV' => false]);
-            $process->inheritEnvironmentVariables();
```

Comment:
this requires bumping the min version of the process component. Allowing 3.4 would not work, as it does not inherit by default.